sqjobs/tests/broker_test.py

```diff
@@ -71,8 +82,14 @@ def test_read_job(self):
         assert jobs[0] == {'args': (2, 2), 'kwargs': {}, 'name': 'adder'}
         assert jobs[1] == {'args': (1, 1), 'kwargs': {}, 'name': 'adder'}
 
+        broker.add_job(Adder, 2, 2)
+        gen = broker.jobs('default', timeout=0)
+        jobs = []
+        jobs.append(next(gen))
+        assert len(jobs) == 1
```

Comment:
This is a little weird. Not sure what you aim to test here, that was not tested in the previous line.